masterbase/anomaly.py

```diff
@@ -61,23 +97,27 @@ def transition_freqs(data: bytes) -> NDArray:
 
 
 class DetectionState(BaseModel):
-    """Accumulate a trace of statistics for determining the likelihood of observed data under Markov transition frequencies present in valid data.
-
-    Args:
+    """
+    Accumulate a trace of statistics for determining the likelihood of observed data under Markov transition 
+    frequencies present in valid data.
+    
+    Attributes:
         length: The cumulative length of all inputs thus far.
```

Comment:
does sphinx not work with google-notypes docstring convention? Or just favoring this style?